jsapp/js/popoverMenu.tsx

```diff
@@ -13,9 +13,9 @@ interface PopoverMenuProps {
   type?: string;
   additionalModifiers?: string[];
   /** the element that will be opening the menu, menu will be placed in relation to it */
-  triggerLabel: JSX.Element;
+  triggerLabel: React.ReactNode;
```

Comment:
Interesting. Should we apply this to helpBubble.tsx as well? 

```patch
diff --git a/jsapp/js/components/support/helpBubble.tsx b/jsapp/js/components/support/helpBubble.tsx
index 295c7437a..cd214813a 100644
--- a/jsapp/js/components/support/helpBubble.tsx
+++ b/jsapp/js/components/support/helpBubble.tsx
@@ -277,7 +277,7 @@ class HelpBubble extends React.Component<{}, HelpBubbleState> {
   }
 
   render() {
-    let popupRenderFn: () => JSX.Element | null = () => null;
+    let popupRenderFn: () => React.ReactNode | null = () => null;
     const modifiers = ['support'];
     if (this.state.isOpen) {
       modifiers.push('open');
```

Reply:
`React.ReactNode` and `JSX.Element` are very similar. And I've learned more about their differences: https://stackoverflow.com/a/72353143/2311247. I think that unless TS complains, there is no need to change these.